week3/src/gamePage/gamePage.css

```
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Jua&display=swap');
+
+body{
+    font-family: 'Jua', sans-serif;
+    margin:0px;
+    overflow-x: hidden;
+}
+
+
+#game-page .header{
+    width:100%;
+    height: 80px;
+    justify-content: center;
+    display:flex;
+    background-color: black;
+    color:whitesmoke;
+    padding: 10px;
+    margin:0px;
+}
+
+#game-page .game-page--main{
+    width:100%;
+    justify-content: center;
+    align-items: center;
+    display:flex;
+    flex-direction: row;
+    margin:0px;
+    background-color: white;
+}
+
+section.content-wrapper{
+    width: 100%;
+    height: 650px;
+    display:flex;
+    justify-items: center;
+    justify-content: center;
+    flex-direction: row;
+}
+
+.content{
+    width: 50%;
+    height: 100%;
+    display: flex;
+    align-items:center;
+    flex-direction: column;
+    position:relative;
+}
+
+.versus-img{
+    width:200px;
+    position: absolute;
+    left:43%;
+    top:40%;
+    z-index: 3;
+}
```

사소한 부분이긴 한데 이런 식으로 스타일링 해주면 position:absolute인 애가 무조건 화면 가운데에 위치하게 되어요 ! 
```suggestion
.versus-img{
    width:200px;
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    z-index: 3;
}
```

---

versus 이미지를 무조건 화면 가운데에 두고 싶다면 저런 식으로 바꿔봐도 좋을 것 같슴다 ㅎㅎ

---

지금은 화면 너비에 따라서 위치가 조금씩 어긋나는 것 같아서요 !!

---

아 그렇군요! 전에 한번 봤던 건데 잊고 있었어용 감사합니다 :>